### ISSUE TITLE
DNS_PROBE_FINISHED_NXDOMAIN error

### ISSUE THREAD

**Issue by jamiematrix** [craftcms/nitro#398]
### Description
I've started getting random sites not responding today. I initially had an issue with a site that wouldn't update from it's repo, so I trashed the local repo and pulled again. After this I wasn't able to connect to the site, so trashed the Nitro host and re-created. No joy. Went to look at another site and that had stopped working too.

Currently, 6 out of the 10 Nitro hosts I have are not working. They all produce the same error when trying to browse. I've tried other browsers and they have the same message. Also tried deleting some of those sites and re-creating, and some I've created with different host names.

As far as I know, all of these sites have worked in the last week. The laptop hasn't installed any OS updates that I'm aware of.

### Steps to reproduce

1. Visit an existing site OR create a new one

### Additional info

- Nitro version: 2.0.8
- Docker version: 20.10.7 using WSL

**Comment by jamiematrix** [comment]
Just removed all but the MySQL and the main host I'm trying to work on from the hosts file and the site loaded.

It's looking like I can only have 8 URLs after 127.0.0.1 and the MySQL host. Any further host URLs after the 8th give the DNS_PROBE error in Chrome.

**Comment by jasonmccallister** [comment]
@jamiematrix that is rather strange... Can you try another browser with more than 8 hosts?

**Comment by jamiematrix** [comment]
I've tried Edge and Firefox, both do the same.

**Comment by jamiematrix** [comment]
I've just split out the alias' onto 2 lines, both starting with 127.0.0.1 and I now have 13 hosts working.

**Comment by jasonmccallister** [comment]
I'm wondering if this is a linux WSL issue... I'm on a macOS system and can't replicate with more than 8 hosts... all of the domains resolve. 

Can you try `curl -v hostname.nitro` and provide the output?

![CleanShot 2021-08-16 at 12 25 47@2x](https://user-images.githubusercontent.com/5354908/129597151-bd0a6b38-539d-416c-813d-49b13ddb07a6.png)

**Comment by jamiematrix** [comment]
The superuser.com post suggests its a Windows issue regardless of WSL. I've had a quick look around and can't see anything to suggest it is just Windows, but there doesn't seem to be anything about an alias limit anywhere (so far).

Running that command I get the source (HTML) of the index page that would load. Does that sound right? There's a lot in there hehe.

**Comment by angrybrad** [comment]
Yup, not a WSL thing, but a Windows one. Just verified that I can only add 9 aliases to a single IP address on my Windows 11 box. We should probably start breaking these up one per domain.

**Comment by jamiematrix** [comment]
Would be nice if it were documented slightly better than stumbling upon it, thanks Microsoft. Glad there is a workaround, and sounds like should be a relative easy fix to output the hosts text?

**Comment by jasonmccallister** [comment]
Hi, we are closing this issue as we have decided to retire Nitro, so no additional work will occur on this project. You can read the official blog post here https://craftcms.com/blog/retiring-craft-nitro. We appreciate everyones feedback and involvement and we look forward to refocusing our efforts on Cloud! 

If you're looking for a new local development environment, we recommend DDEV and have a knowledge base article to help you with the transition: https://craftcms.com/knowledge-base/migrating-from-craft-nitro-to-ddev.
